gradle-plugin/plugin/build.gradle.kts

```diff
@@ -126,12 +145,14 @@ dependencies {
   implementation(libs.kotlinx.serialization)
   implementation(libs.okhttp)
 
+  testImplementation(libs.slf4j.api)
```

Comment:
This was causing compilation issues since we directly use the logger api to pass to the transform in one of the tests. Not sure why this wasn’t an issue before.